fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -719,10 +805,13 @@ ACTOR Future<Void> refreshEncryptionKeysImpl(Reference<EncryptKeyProxyData> ekpP
 		}
 
 		ekpProxyData->baseCipherKeysRefreshed += rep.cipherKeyDetails.size();
-
+		ekpProxyData->kmsConnectorHealthStatus = KMSHealthStatus{ true, true, now() };
```

Comment:
we still want to check if response `vector.size > 1` before updating the KmsHealthStatus?